docs/specs/code/SERVER_STATUS_ENDPOINT.md

```diff
@@ -0,0 +1,37 @@
+# Abstract
+
+Describe a new endpoint to notify server issues in Yoroi-frontend.
+   
+# Motivation
+
+Currently, Yoroi-frontend has no way to notify neither server issues nor connection issues. Also, once Yoroi has the ability to notify server issues, the endpoint used by Yoroi to fetch the information can be used to create a public URL that reports the mentioned status, such as [AWS](https://status.aws.amazon.com/) and [Twitter](https://status.twitterstat.us).
+
+# Proposal
+
+Adding a new endpoint in the backend named `api/status` is proposed. This endpoint has the method `GET` that returns the last JSON inserted in the following table.
```

Comment:
This talks about the last inserted JSON in the table, which seems a little bit odd, because what happens once the outbreak is done? there isn’t a way to turn it off; the only way is deleting it. So it’s expected that it will probably just return the only message that should be in the table (I’m not particularly pushing for this alternative, but rather it’s what I deduct from the implementation)

Reply:
I'll clarify in the spec that when the outbreak is done, a new JSON have to be inserted indicating that the server is ok.